### DISCUSSION TITLE
Posting questions related to using Couenne in Minizinc

### DISCUSSION THREAD

**Discussion by indrajitsg** [coin-or/Couenne#52] [General]
Hi All,

I have been playing around with Minizinc for a while now. And time to time, I test out my optimization problems using various solvers that are downloadable from the AMPL website. I was wondering if it is okay to post issues or have discussions around the issues that I observe in using Minizinc + Couenne?

Regards,
Indrajit

**Comment by merraksh** [comment]
Hi Indrajit,

 

yes, feel free.

 

Best,

Pietro
 

Sent: Monday, January 11, 2021 at 5:52 AM

**Comment by tkralphs** [comment]
But please note that this mailing list is being retired and you should use
Issues or Discussions (as appropriate) on Github going forward.

Cheers,

Ted

On Mon, Jan 11, 2021 at 10:06 AM Pietro Belotti <notifications@github.com>
wrote:

> Hi Indrajit,
>
>
>
> yes, feel free.
>
>
>
> Best,
>
> Pietro
>
>
> Sent: Monday, January 11, 2021 at 5:52 AM
> From: "Indrajit Sen Gupta" <notifications@github.com>
> To: "coin-or/Couenne" <Couenne@noreply.github.com>
> Cc: "Subscribed" <subscribed@noreply.github.com>
> Subject: [coin-or/Couenne] Posting questions related to using Couenne in
> Minizinc (#52)
>
>
>
>
> Hi All,
>
> I have been playing around with Minizinc for a while now. And time to
> time, I test out my optimization problems using various solvers that are
> downloadable from the AMPL website. I was wondering if it is okay to post
> issues or have discussions around the issues that I observe in using
> Minizinc + Couenne?
>
> Regards,
> Indrajit
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coin-or/Couenne/discussions/52#discussioncomment-274590>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABFWVJUVZTLNQDOKSFZ4M2TSZMHWFANCNFSM4V5AWAWA>
> .
>


-- 
Dr. Ted Ralphs
Professor, Industrial and Systems Engineering
Lehigh University
(610) 628-1280
ted 'at' lehigh 'dot' edu
coral.ie.lehigh.edu/~ted

**Reply by indrajitsg** [reply]
Hey Ted, this message has been posted on Github discussions only.

Regards,
Indrajit
